dotCMS/src/main/java/com/dotcms/rest/RestClientBuilder.java

```diff
@@ -19,13 +19,19 @@
  */
 public class RestClientBuilder {
 
+    private RestClientBuilder() {}
+
+    private static class RestClientLazyHolder {
+        static final Client REST_CLIENT = newClient();
+    }
+
 	/**
 	 * Creates a new instance of the REST client used to access the RESTful
 	 * services available in the dotCMS back-end.
 	 * 
 	 * @return The REST {@link Client} object.
 	 */
-    public static Client newClient() {
+    private static Client newClient() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this method into "RestClientLazyHolder". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3398)